src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/project.json

```diff
@@ -1,8 +1,4 @@
 {
-  "supports": {
```

Comment:
I'm not against adding these if we do so systematically for all projects and it won't cause nuget to spew warnings.